jobs/client_count_daily_view.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [[ -z "$bucket" || -z "$date" ]]; then
+    echo "Missing arguments!" 1>&2
+    exit 1
+fi
+
+git clone https://github.com/mozilla/telemetry-batch-view.git
+cd telemetry-batch-view
+sbt assembly
+
+base="normalized_channel,"
+base+="country,"
+base+="locale,"
+base+="app_name,"
+base+="app_version,"
+base+="e10s_enabled,"
+base+="e10s_cohort,"
+base+="os,"
+base+="os_version"
+
+select="substr(subsession_start_date, 0, 10) as activity_date,"
```

Comment:
Per [Bug 1414281](https://bugzilla.mozilla.org/show_bug.cgi?id=1414281), let's be more selective about `activity_date`:
`regexp_extract(subsession_start_date, '(20[0-9]{2}-[0-9]{2}-[0-9]{2})', 1) as activity_date`

This implies also adding `AND activity_date IS NOT NULL` to the `--where` clause (which wouldn't work right if this were SQL, but it looks like it should work here -- please test)

Reply:
tested, this worked for `submission_date=20171101`